utils/build-script-impl

```diff
@@ -241,11 +241,11 @@ function set_deployment_target_based_options() {
                     cmark_cmake_options=(
                         -DCMAKE_C_FLAGS="$(cmark_c_flags $deployment_target)"
                         -DCMAKE_CXX_FLAGS="$(cmark_c_flags $deployment_target)"
-                        -DCMAKE_OSX_SYSROOT:PATH="$(xcrun --sdk $xcrun_sdk_name --show-sdk-path)"
+                        -DCMAKE_OSX_SYSROOT:PATH=$(printf %q "$(xcrun --sdk $xcrun_sdk_name --show-sdk-path)")
```

Comment:
While the paths are quoted here, they are subsequently used in other locations where they are inconsistently quoted.  Escaping whitespace in the path itself via printf %q resolved the related build issues I observed.